js/main.js

```diff
@@ -18,8 +21,96 @@ $entryForm.addEventListener('submit', function (event) {
     photoUrl: $entryForm.photo.value,
     note: $entryForm.notes.value,
   };
+
+  $ul.prepend(renderEntry(newEntry));
+
+  viewSwap('entries');
+  toggleNoEntries();
+
   data.nextEntryId++;
   data.entries.unshift(newEntry);
   $img.setAttribute('src', 'images/placeholder-image-square.jpg');
   $entryForm.reset();
 });
+
+// Render Entry Function
+
+function renderEntry(entry) {
+  const $newListItem = document.createElement('li');
+  $newListItem.setAttribute('class', 'new-list-item');
+
+  const $newRow = document.createElement('div');
+  $newRow.setAttribute('class', 'row');
+
+  const $entryNew = document.createElement('div');
+  $entryNew.setAttribute('class', 'column-half');
+
+  const $imgNew = document.createElement('img');
+  $imgNew.setAttribute('src', entry.photoUrl);
+
+  const $newText = document.createElement('div');
+  $newText.setAttribute('class', 'column-half');
+
+  const $newTitle = document.createElement('h3');
+  $newTitle.textContent = entry.title;
+
+  const $newNote = document.createElement('p');
+  $newNote.textContent = entry.note;
+
+  $newListItem.appendChild($newRow);
+
+  $newRow.appendChild($entryNew);
+  $entryNew.appendChild($imgNew);
+
+  $newRow.appendChild($newText);
+  $newText.appendChild($newTitle);
+  $newText.appendChild($newNote);
+
+  return $newListItem;
+}
+
+document.addEventListener('DOMContentLoaded', function (event) {
+  const $uList = document.getElementById('entries-list');
```

Comment:
You already have a `querySelector` for this on line 13. You should just utilize that variable